--- compiz-0.9.1.orig/plugins/scale/src/scale.cpp	2017-01-09 17:08:36.000000000 +0200
+++ compiz-0.9.1/plugins/scale/src/scale.cpp	2017-12-30 11:44:26.496954937 +0200
@@ -1267,6 +1267,18 @@
     {
 	SCALE_SCREEN (::screen);
 
+        //find client leader for scale group
+        CompWindow *w;
+        Window     cL;
+        cL = (Window) CompOption::getIntOptionNamed (options, "window");
+        w = ::screen->findWindow (cL);
+        if (w)
+          ss->priv->clientLeader = (w->clientLeader ()) ?
+              w->clientLeader () : cL;
+        else
+          ss->priv->clientLeader = None;
+        //
+
 	if (ss->priv->actionShouldToggle (action, state) &&
 	    (ss->priv->state == ScaleScreen::Wait ||
 	     ss->priv->state == ScaleScreen::Out))
