diff --git src/vs/platform/telemetry/common/telemetryService.ts src/vs/platform/telemetry/common/telemetryService.ts
index 23eab27567..f727410070 100644
--- src/vs/platform/telemetry/common/telemetryService.ts
+++ src/vs/platform/telemetry/common/telemetryService.ts
@@ -149,7 +149,7 @@ Registry.as<IConfigurationRegistry>(Extensions.Configuration).registerConfigurat
 		'telemetry.enableTelemetry': {
 			'type': 'boolean',
 			'description': localize('telemetry.enableTelemetry', "Enable usage data and errors to be sent to Microsoft."),
-			'default': true
+			'default': false
 		}
 	}
-});
\ No newline at end of file
+});
