diff -Naur vscode-1.20.1.orig/src/vs/workbench/parts/update/electron-browser/update.contribution.ts vscode-1.20.1/src/vs/workbench/parts/update/electron-browser/update.contribution.ts
--- src/vs/workbench/parts/update/electron-browser/update.contribution.ts	2018-02-13 17:06:40.000000000 +0200
+++ src/vs/workbench/parts/update/electron-browser/update.contribution.ts	2018-03-01 07:15:01.279906451 +0200
@@ -55,7 +55,7 @@
 		'update.channel': {
 			'type': 'string',
 			'enum': ['none', 'default'],
-			'default': 'default',
+			'default': 'none',
 			'description': nls.localize('updateChannel', "Configure whether you receive automatic updates from an update channel. Requires a restart after change.")
 		},
 		'update.enableWindowsBackgroundUpdates': {
